plugins/fcp_can_c/fcp_can_c/can_c_writer.py

```diff
@@ -279,7 +270,20 @@ def initialize_can_data(
         period = extension.fields.get("period", -1)
 
         if not any(node.name == device_name for node in devices):
-            devices.append(CanNode(device_name))
+            rpc_get_id = None
+            rpc_ans_id = None
+            for dev in getattr(fcp, "devices", []):
```

Comment:
```suggestion
            for dev in fcp.devices:
```

devices must exist in fcp